reflex-dom-core/src/Reflex/Dom/Builder/Immediate.hs

```diff
@@ -822,7 +913,7 @@ hoistTraverseWithKeyWithAdjust base mapPatch updateChildUnreadiness applyDomUpda
                   when (DMap.null newUnready) $ do
                     applyDomUpdate p
   (children0, children') <- ImmediateDomBuilderT $ lift $ base (\k v -> drawChildUpdate initialEnv markChildReady $ f k v) dm0 dm'
-  let processChild k (Compose (_, _, sRef, _)) = ComposeMaybe <$> do
+  let processChild k (_child_readyState . getCompose -> sRef) = ComposeMaybe <$> do
```

Comment:
yeah I had more uses but then backed off from them. I'll clean out this case to not have a one-off